cwms-servers/CWMS_HOME/bin/logWatcher.py

```diff
@@ -1,8 +1,14 @@
+#!/bin/env python
+
 import os
 import time
 import re
 import logging
 import argparse
+import sys
+cwms_home = os.getenv('CWMS_HOME')
+sys.path.append("{}/bin/lib".format(cwms_home))
+sys.path.append("{}/bin".format(cwms_home))
```

Comment:
if not done further on in the script, echo the path out ?

Reply:
updated